wapitiCore/wappalyzer/wappalyzer.py

```diff
@@ -307,6 +332,10 @@ def detect_application_versions(self, application: dict) -> Set[str]:
             "html": self._html_code,
             "scriptSrc": self._scripts,
         }
+
+        # Detect version of dom element
+        versions.update(self.detect_versions_normalize_dom(application))
```

Comment:
Can you move this before defining `elements_to_check` or after the last `versions.update` call ?